x/crisis/internal/keeper/keeper.go

```diff
@@ -83,4 +83,14 @@ func (k Keeper) AssertInvariants(ctx sdk.Context) {
 	logger.Info("asserted all invariants", "duration", diff, "height", ctx.BlockHeight())
 }
 
+// InvCheckPeriod returns the invariant checks period.
+func (k Keeper) InvCheckPeriod() uint { return k.invCheckPeriod }
+
+// FeeCollectorName returns the name of the FeeCollector module.
+func (k Keeper) FeeCollectorName() string { return k.feeCollectorName }
+
+func (k Keeper) SendCoinsFromAccountToModule(ctx sdk.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) sdk.Error {
```

Comment:
The only valid execution of this is to send from an account to `k.FeeCollectorName()` _only_. So I'm thinking we change this to:

```go
func (k Keeper) SendCoinsFromAccountToModule(ctx sdk.Context, senderAddr sdk.AccAddress, amt sdk.Coins) sdk.Error {
  return k.supplyKeeper.SendCoinsFromAccountToModule(ctx, senderAddr, k.feeCollectorName, amt)
}
```